library/CMakeLists.txt

```diff
@@ -273,6 +279,7 @@ package_set_postinst_prerm(
     "hiprand;rocrand"
     "${CMAKE_INSTALL_PREFIX}/hiprand/lib;${CMAKE_INSTALL_PREFIX}/rocrand/lib"
     "${CMAKE_INSTALL_PREFIX}/hiprand/include;${CMAKE_INSTALL_PREFIX}/rocrand/include"
+    "/opt/rocm/hiprand/lib;/opt/rocm/rocrand/lib"
```

Comment:
Is /opt/rocm ok to use here?

Reply:
Yes. This parameter is used to write the ld.conf.so entry, which is preferred to have it as /opt/rocm. So any external application/framework want to use this lib, will be able to find it through ldconf entry with the symlink to the required version of rocm.

Reply:
I am not sure about this. It seems we are hard coding it. Also, this will not work with SPACK. We should no assume that anything exists in /opt/rocm

Reply:
Yes, it is hard coding. As per the current design the loader will search all the dependent libs based on RUNPATH using $ORIGIN. It will not use ldconf entries or LD_LIBRARY_PATH, as per the customer request. The above changes are required for normal users who have /opt/rocm symlink and they will use the ldconf entries to load the required libraries.